dotnet/Microsoft.MacCatalyst.Templates/maccatalyst/MacCatalystApp1.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-maccatalyst</TargetFramework>
+    <RuntimeIdentifier>maccatalyst-x64</RuntimeIdentifier>
+    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">MacCatalystApp1</RootNamespace>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
```

Comment:
This template seems like it works, do I *need* a provisioning profile for catalyst?

    Could not find any available provisioning profiles for foo on MacCatalyst.